go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -156,6 +162,9 @@ func (bh *S3BackupHandle) AddFile(ctx context.Context, filename string, filesize
 			Key:                  object,
 			Body:                 reader,
 			ServerSideEncryption: sseOption,
+			SSECustomerAlgorithm: bh.bs.s3SSECustomer.alg,
```

Comment:
Just to be sure, is it safe to pass these 3 fields as nil? Since that is what will happen when this functionality is not being used.

Reply:
Yes, these values are should be nil in case they are not used.

Reply:
Yeah the aws sdk handles the cases where these are nil, since not everyone even sets these values on the input parameter.